cmd/utils.go

```diff
@@ -95,10 +95,17 @@ func ParseFlags(scenarioDetail *models.ScenarioDetail, args []string, scenarioCo
 				return nil, nil, err
 			}
 		}
-		if value != nil {
-			environment[*field.Variable] = *value
-			if field.Type == typing.File {
-				volumes[*field.MountPath] = *value
+		if value != nil && *value != "" {
+			if field.Type != typing.File {
+				environment[*field.Variable] = *value
+			} else if field.Type == typing.File {
+				if field.MountPath != nil {
+					volumes[*value] = *field.MountPath
+					environment[*field.Variable] = *field.MountPath
+				} else {
+					fileSrcDst := strings.Split(*value, ":")
```

Comment:
@tsebastiani I can take this out or leave as a backup, what do you think?

Reply:
I'd say that is better to remove it, I don't like the idea that someone can arbitrary decide to override a file inside the container